size/sizes.go

```diff
@@ -19,28 +19,30 @@ import (
 )
 
 func init() {
-	var a bool
-	SizeOfBool = int(reflect.TypeOf(a).Size())
-	var b float32
-	SizeOfFloat32 = int(reflect.TypeOf(b).Size())
-	var c float64
-	SizeOfFloat64 = int(reflect.TypeOf(c).Size())
-	var d map[int]int
-	SizeOfMap = int(reflect.TypeOf(d).Size())
-	var e *int
-	SizeOfPtr = int(reflect.TypeOf(e).Size())
-	var f []int
-	SizeOfSlice = int(reflect.TypeOf(f).Size())
-	var g string
-	SizeOfString = int(reflect.TypeOf(g).Size())
-	var h uint8
-	SizeOfUint8 = int(reflect.TypeOf(h).Size())
-	var i uint16
-	SizeOfUint16 = int(reflect.TypeOf(i).Size())
-	var j uint32
-	SizeOfUint32 = int(reflect.TypeOf(j).Size())
-	var k uint64
-	SizeOfUint64 = int(reflect.TypeOf(k).Size())
+	var b bool
+	SizeOfBool = int(reflect.TypeOf(b).Size())
+	var f32 float32
+	SizeOfFloat32 = int(reflect.TypeOf(f32).Size())
+	var f64 float64
+	SizeOfFloat64 = int(reflect.TypeOf(f64).Size())
+	var i int
+	SizeOfInt = int(reflect.TypeOf(i).Size())
+	var m map[int]int
+	SizeOfMap = int(reflect.TypeOf(m).Size())
```

Comment:
SizeOfMap implies something more general, but is it really the same regardless of type?

Reply:
Yes, the overhead is the same regardless of the types the map contains.

Reply:
thanks, in that case i can simplify my change as well and use this